charts/traces/templates/tests/test-traces.yaml

```diff
@@ -1,24 +1,4 @@
 apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: queries
-data:
-  # Queries have the format <JMESPath query> -> <matching regexp>.
-  # This syntax is based on the syntax used in the JMESPath docs, which describe queries as:
-  # search(query, input) -> result
-  queries: |
-    Path -> ^/v1/otel/v1/traces$
-
----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: cluster-info
-data:
-  id: "fake-cluster-id"
```

Comment:
known regression: this (traces) test will now fail if you run it in isolation, as it depends on the `events` pod (which manages the creation of this configmap) of the `stack` test having run first. Luckily it is normally not run in isolation, as both `make test` and the github test automation run the `stack` and then `traces` test in order.

There is another open issue to have `traces` run its own init container to set up cluster-info, which will fix this regression.